code-of-conduct.md

```diff
@@ -0,0 +1,96 @@
+
+# The Short Version
+
+Be respectful of other people; respectfully ask people to stop if you are bothered; respect privacy; understand we're mostly not-for-profit; and if you can’t resolve an issue then contact the Administrators. If you are being a problem, it will be apparent and you may be asked to leave the Soft Skills Engineering Slack.
+
+# The Long Version
+
+## Respect
+
+The Soft Skills Engineering Slack is an intentionally positive community that recognizes and celebrates the creativity and collaboration of independent members and the diversity of skills, talents, experiences, cultures, and opinions that they bring to our community.
+
+The Soft Skills Engineering Slack is an inclusive environment, based on treating all individuals respectfully, regardless of gender or gender identity (including transgender status), sexual orientation, age, disability, nationality, ethnicity, religion (or lack thereof), or career path.
+
+We value respectful behavior above individual opinions.
+
+Respectful behavior includes but is not limited to:
+
+* Be considerate, kind, constructive, and helpful.
+* Avoid demeaning, discriminatory, harassing, hateful, or physically threatening behavior, speech, and imagery.
+* If you’re not sure, ask someone instead of assuming. No, really. Just ask the administrators. We’d rather hear from you than hear about something you said or did after the fact, and we are here to help.
+* Don’t be a bystander. Role model respectful behaviour, but also help to address disrespect when you see it. 
+
+Disrespectful behavior outside this community may be considered a violation of this code of conduct at the discretion of the Administrators.
+
+## Privacy
+
+This community is not a public space. However, no one has signed an non-disclosure agreement (“NDA”) to participate, and you should not presume anything you say here will remain private, so act accordingly. Protect IP and legally-protected information.
+
+If you want to publicly disclose anything discussed here, use the [Chatham House Rule](https://www.chathamhouse.org/about/chatham-house-rule) as the guideline (“participants are free to use the information received, but neither the identity nor the affiliation of the speaker(s), nor that of any other participant, may be revealed”).
+
+For attribution of specific content found on this Slack, we ask that you ask the originator of the content for permission. If you don’t receive consent in a reasonable period of time, we ask that you credit the “Soft Skills Engineering Slack.”
+
+## Not For Profit
+
+This community is mostly not a place for obvious commercial activity such as recruiting or marketing except in channels dedicated to that purpose which include:
+
+* #conferences
+* #consulting-gigs
+* #jobs
+* #remote-jobs
+* #services
```

Comment:
We don't have these channels, so I recommend we revise this section.

Reply:
I was thinking of just adding them, it's pretty common to have these on the community slack teams I hang out in